src/test/java/de/tum/in/www1/artemis/util/CourseTestService.java

```diff
@@ -669,6 +669,26 @@ else if (submission instanceof ModelingSubmission modelingSubmission) {
         }
     }
 
+    // Test
```

Comment:
Why do we need this comment? Below as well

Reply:
This is a general convention for us, to clarify that this is more or less a @Test method, that is called once for BitBucketBamboo and once for GitLabJenkins